src/openrct2/windows/GameBottomToolbar.cpp

```diff
@@ -348,14 +369,32 @@ static void window_game_bottom_toolbar_paint(rct_window *w, rct_drawpixelinfo *d
         w->y + window_game_bottom_toolbar_widgets[WIDX_RIGHT_OUTSET].bottom,
         PALETTE_51
     );
+    if (theme_get_flags() & UITHEME_FLAG_USE_FULL_BOTTOM_TOOLBAR)
+    {
+        // Draw grey background
```

Comment:
This part is always grey, no matter what the colour of the bottom toolbar is. Disable the check and the rct1 bottom toolbar and see.